app/components/HeaderNotifications/index.tsx

```diff
@@ -56,68 +45,94 @@ const NotificationElement = ({
   return null;
 };
 
-export default class NotificationsDropdown extends Component<Props, State> {
-  state = {
-    notificationsOpen: false,
-  };
-  fetch = () => {
-    this.props.fetchNotifications();
-    this.props.fetchNotificationData();
-  };
-  renderNotifications = (notifications: Array<Record<string, any>>) => {
-    return (
-      <Dropdown.List className={styles.maxHeight}>
-        {notifications.map((notification) => (
-          <Dropdown.ListItem key={notification.id}>
-            <ErrorBoundary hidden>
-              <NotificationElement notification={notification} />
-            </ErrorBoundary>
-          </Dropdown.ListItem>
-        ))}
-      </Dropdown.List>
-    );
+type HeaderNotificationsConentProps = {
+  notifications: AggregatedActivity[];
+  fetchingNotifications: boolean;
+};
+
+const HeaderNotificationsConent = ({
```

Comment:
I assume this is supposed to say Content? Likewise for line 48

Reply:
Yes - nicely spotted!